src/main/java/graphql/schema/GraphQLTypeRefResolvingVisitor.java

```diff
@@ -0,0 +1,44 @@
+package graphql.schema;
+
+import graphql.Internal;
+import graphql.util.TraversalControl;
+import graphql.util.TraverserContext;
+
+@Internal
+public class GraphQLTypeRefResolvingVisitor extends GraphQLTypeVisitorStub {
```

Comment:
what do you think about putting this class into the other one `GraphQLTypeResolvingVisitor`? it is a very hard dependency and it would make it more readable imho